modules/core/src/test/java/org/apache/ignite/internal/processors/cache/CacheGroupsMetricsRebalanceTest.java

```diff
@@ -173,11 +173,13 @@
     }
 
     /**
+     * Checking the correctness of {@link CacheMetrics#getRebalancingKeysRate}.
```

Comment:
We try to start all method description from a verb with "s" in the tail everywhere.